build.gradle

```diff
@@ -408,7 +408,7 @@ jar {
 jib {
     extraDirectories.paths = ['build/gen-expanded']
     from {
-        image = "gcr.io/distroless/java-debian10:11"
+        image = "gcr.io/distroless/java11-debian11"
```

Comment:
I recommend we switch to `us.gcr.io/broad-dsp-gcr-public/base/jre:11-distroless`, the appsec trusted version of the distroless image.

Reply:
good call

Reply:
That image doesn't seem to work (at least yet...a newer version might be needed).  I'm going to stick with `gcr.io/distroless/java11-debian11` for now